bigquery_etl/view/generate_stable_views.py

```diff
@@ -241,6 +217,25 @@ def get_stable_table_schemas() -> List[SchemaFile]:
     ]
 
 
+def schemas_uri_for_project(target_project):
+    """Return URI for the schemas tarball deployed to the given project.
+
+    We construct a fake query and send it to the dry run service in order
+    to read dataset labels, which contains the commit hash associated
+    with the most recent production schemas deploy.
+    """
+    with tempfile.TemporaryDirectory() as tdir:
+        d = Path(tdir) / target_project / "telemetry_derived" / "myquery"
+        d.mkdir(parents=True)
+        tfile = Path(d) / "query.sql"
+        with tfile.open("w") as f:
+            f.write("SELECT 1")
```

Comment:
This section is removed by my other suggestion, but `Path` has `write_text` and `write_bytes` methods, so this could done with:
```suggestion
        tfile.write_text("SELECT 1")
```